test/integration/functional_test.go

```diff
@@ -316,6 +337,26 @@ func validateImageCommands(ctx context.Context, t *testing.T, profile string) {
 		checkImageExists(ctx, t, profile, taggedImage)
 	})
 
+	// docs: Try to load image already loaded and make sure `minikube image load --daemon` works
+	t.Run("ImageReloadDaemon", func(t *testing.T) {
+		rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "image", "load", "--daemon", taggedImage))
```

Comment:
did you mean this to be true ? lets explicitly set the true or false here

--daemon="TrueOrFalse"

Reply:
Putting a `bool` flag without a value defaults to `true`, for example `--force`, you don't have to put `--force=true`, it will default to true.

Reply:
Yeah meant to be true, wanted to follow the style of all the other tests written in this file.